mr/src/main/java/org/apache/iceberg/mr/InputFormatConfig.java

```diff
@@ -49,6 +49,9 @@ private InputFormatConfig() {
   public static final String SERIALIZED_TABLE_PREFIX = "iceberg.mr.serialized.table.";
   public static final String TABLE_CATALOG_PREFIX = "iceberg.mr.table.catalog.";
   public static final String LOCALITY = "iceberg.mr.locality";
+  public static final String IDENTIFIER_FIELD_NAMES = "iceberg.identifier-field-names";
+  // Usually, column names containing ',' are not supported by hive, so we can use ',' as separator.
```

Comment:
AFAIK with the correct escaping, you can have anything in the column names. Could you please try that out?
Thanks, Peter

Reply:
We work around this when pushing column names with another config which allows us to provide a separator too. This could default to `, `,  so it is rarely used, but allows the user to fix the schema, if needed